config.json.example

```diff
@@ -33,7 +33,8 @@
     "gratitude": "000",
     "staffaccountability": "000",
     "commandcenter": "000",
-    "contentNotifier": "000"
+    "contentNotifier": "000",
+	"errors:" "000"
```

Comment:
Would you be able to match indentation here?

Reply:
Just have to say this trips me up every time, this file uses spaces and everything else Horace-related uses tabs. Not sure if that's something we want to take action on or not.

Reply:
That's a good point. Honestly I would rather switch to a different type of config file, because it feels like we've outgrown JSON, but we'll figure something out.